components/script/script_thread.rs

```diff
@@ -3401,6 +3405,20 @@ impl ScriptThread {
         unsafe { document.handle_wheel_event(wheel_delta, point, node_address, can_gc) };
     }
 
+    /// Handle scroll event
```

Comment:
I think just remove this doc comment that simply repeats the name of the method.